Vishnu_45/html/Assignment3/ass2_contact.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
need proper code indentation
remove all extra lines
add new line
change all with at the end of css property ends
remove  !DOCTYPE html